test/e2e/manifest.yaml

```diff
@@ -0,0 +1,29 @@
+# Test manifest for https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external
+ShortName: manila
+StorageClass:
+  FromExistingClassName: standard-csi
```

Comment:
This will probably pick up Cinder storage class for the tests. You should use name of existing Manila StorageClass to be used as template of storage classes used for testing.

Another option is `FromName: true`, which will create an empty StorageClass, only with the right `provisioner:` and `volumeBindingMode: Immediate` - if that's enough for Manila.

Reply:
Thanks, I'll add that and see how that works.